deploifai/mlflow/get_setup.py

```diff
@@ -0,0 +1,93 @@
+import click
+import pyperclip
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import pass_deploifai_context_obj, DeploifaiContextObj
+from PyInquirer import prompt
+
+
+@click.command('get-setup')
+@click.option("--external", is_flag=True, help="Use code externally")
+@pass_deploifai_context_obj
+def get_setup(context: DeploifaiContextObj, external):
+    """
+    Command to obtain code for mlflow integration
```

Comment:
Simplify the doc string a little. Try `Get setup to integrate mlflow in training scripts`.

Reply:
Changed